app/(gcforms)/[locale]/(form administration)/form-builder/[id]/components/dialogs/MoreDialog/StrictValue.tsx

```diff
@@ -0,0 +1,47 @@
+import { useTranslation } from "@i18n/client";
+import { Checkbox } from "@formBuilder/components/shared/MultipleChoice";
+import { FormElement } from "@lib/types";
+
+export const StrictValue = ({
+  item,
+  setItem,
+}: {
+  item: FormElement;
+  setItem: (item: FormElement) => void;
+}) => {
+  const { t } = useTranslation("form-builder");
+
+  // ⚠️ Early return if not combobox
+  if (item.type !== "combobox") {
```

Comment:
could use the enum `FormElementTypes.combobox`

Reply:
https://github.com/cds-snc/platform-forms-client/pull/5572/commits/000d3733cf2fcebe32e84a589f84651598719690